torch/_inductor/codegen/cuda/__init__.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import sys
+import tempfile
+
+from typing import List
+
+import torch
+from ...config import cuda as inductor_cuda_config
+
+log = logging.getLogger(__name__)
+
+HAS_CUTLASS = False
+
+_CUTLASS_PY_FULL_PATH = os.path.join(
+    inductor_cuda_config.cutlass_dir, "tools/library/scripts"
```

Comment:
config values can change after pytorch is imported, so we shouldn't do this in global scope

Reply:
Yeah makes sense, will fix this.